__tests__/app.test.js

```diff
@@ -86,18 +84,45 @@ describe("2 GET reviews by id /api/reviews/:review_id", () => {
       .get(`/api/reviews/${review_id}`)
       .expect(404)
       .then(({ body }) => {
-        expect(body.length === 0)
-        expect(body).toEqual({"msg": "No review found for review_id:1000"});
+        expect(body).toEqual({ msg: "No review found for review_id:1000" });
       });
   });
   it("status 400, should return an error", () => {
-    const review_id = 'banana';
+    const review_id = "banana";
     return request(app)
       .get(`/api/reviews/${review_id}`)
       .expect(400)
       .then(({ body }) => {
-        expect(body.length === 0)
-        expect(body).toEqual({"msg": "invalid id type"});
+        expect(body).toEqual({ msg: "invalid id type" });
+      });
+  });
+});
+describe("3 GET users /api/users", () => {
+  it("status 200, should confirm a return of categories containing a description and a slug", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toEqual(4);
+        expect(
+          body.forEach((user) => {
+            expect(user).toEqual(
+              expect.objectContaining({
+                avatar_url: expect.any(String),
+                name: expect.any(String),
+                username: expect.any(String),
+              })
+            );
+          })
+        );
+      });
+  });
+  it("status 404, should ", () => {
+    return request(app)
+      .get("/api/banana")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({});
```

Comment:
perhaps more sensible to test the error message returned, as opposed to the possible emptiness of one's body